include/swift/Frontend/Frontend.h

```diff
@@ -309,6 +309,18 @@ class CompilerInvocation {
     }
     return SourceFile::ImplicitModuleImportKind::Stdlib;
   }
+
+  /// Performs input setup common to these tools:
+  /// sil-opt, sil-func-extractor, sil-llvm-gen, and sil-nm.
+  /// Return value includes the buffer so caller can keep it alive.
+  llvm::ErrorOr<std::unique_ptr<llvm::MemoryBuffer>>
+  setUpInputForSILTool(const llvm::StringRef InputFilename,
+                       const llvm::StringRef ModuleNameArg,
```

Comment:
No `llvm::` and no `const`, please. (You can see the full list of LLVM types we think are common enough to merit this in swift/Basic/LLVM.h)

Reply:
OK.